### PR TITLE
Fix missing headings everywhere

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

Fixes missing headings ToC tags in all pages. 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/docs/apis/realtime/+page.markdoc [hunk 1]
```diff
@@ -109,7 +109,7 @@ More information and examples of authenticating users can be found in the dedica
 # Examples {% #examples %}
 The examples below will show you how you can use Realtime in various ways.
 
-## Subscribe to a Channel
+## Subscribe to a Channel {% #subscribe-to-one-channel %}
```
Comment (eldadfux) [inline src/routes/docs/apis/realtime/+page.markdoc:112]: Can we keep the hashes 1:1 with titles? we're not consistent in many places.

File: src/routes/docs/advanced/platform/permissions/+page.markdoc [hunk 2]
```diff
@@ -63,7 +63,7 @@ The examples below will show you how you can use the different Appwrite permissi
 
 The following examples are using the [Appwrite Web SDK](https://github.com/appwrite/sdk-for-web) but can be applied similarly to any of the other [Appwrite SDKs](/docs/sdks).
 
-## Example #1 - Basic usage
+## Example 1 - Basic usage {% #example-1 %}
```
Comment (gewenyu99) [inline src/routes/docs/advanced/platform/permissions/+page.markdoc:66]:
```suggestion
## Example 1 - Basic usage {% #example-1-basic-usage %}
```

File: src/routes/docs/advanced/platform/permissions/+page.markdoc [hunk 3]
```diff
@@ -96,7 +96,7 @@ promise.then(function (response) {
 });
 ```
 
-## Example #2 - Team roles
+## Example 2 - Team roles  {% #example-2 %}
```
Comment (gewenyu99) [inline src/routes/docs/advanced/platform/permissions/+page.markdoc:99]:
```suggestion
## Example 2 - Team roles  {% #example-2-team-roles %}
```

File: src/routes/docs/advanced/self-hosting/tls-certificates/+page.markdoc [hunk 4]
```diff
@@ -39,7 +39,7 @@ If you want to generate a certificate for a specific domain, pass it as a parame
 docker compose exec appwrite ssl domain="api.myapp.com"
 ```
 
-# Development environment and localhost
+# Development and localhost {% #development %}
```
Comment (gewenyu99) [inline src/routes/docs/advanced/self-hosting/tls-certificates/+page.markdoc:42]:
```suggestion
# Development and localhost {% #development-and-localhost %}
```

File: src/routes/docs/apis/realtime/+page.markdoc (hunk 1 above)
Comment (gewenyu99) [inline src/routes/docs/apis/realtime/+page.markdoc:112]:
```suggestion
## Subscribe to a Channel {% #subscribe-to-a-channel %}
```

File: src/routes/docs/apis/realtime/+page.markdoc [hunk 5]
```diff
@@ -503,10 +503,13 @@ client.setEndpointRealtime("wss://cloud.appwrite.io/v1/realtime")
...
+## Subscription changes {% #subscription-changes %}
 
-The SDK creates a single WebSocket connection for all subscribed channels. Each time a channel is added or unsubscribed - the SDK currently creates a completely new connection and terminates the old one. Therefore, subscriptions to channels should always be done in conjunction with state management so as not to be unnecessarily built up several times by multiple components' life cycles.
+The SDK creates a single WebSocket connection for all subscribed channels. 
+Each time a channel is added or unsubscribed, the SDK currently creates a completely new connection and terminates the old one. 
+Therefore, subscriptions to channels should always be done in conjunction with state management so as not to be unnecessarily 
+built up several times by multiple components' life cycles.
 
-## Server-side SDKs
+## Server SDKs {% #server-sdk %}
```
Comment (gewenyu99) [inline src/routes/docs/apis/realtime/+page.markdoc:513]:
```suggestion
## Server SDKs {% #server-sdks %}
```

File: src/routes/docs/products/auth/quick-start/+page.markdoc [hunk 6]
```diff
@@ -168,7 +168,7 @@ mutation {
 ```
 {% /multicode %}
 
-# More ways to Authenticate
+# More ways to authenticate {% #more-authentication %}
```
Comment (gewenyu99) [inline src/routes/docs/products/auth/quick-start/+page.markdoc:171]:
```suggestion
# More ways to authenticate {% #more-ways-to-authenticate %}
```

File: src/routes/docs/products/databases/databases/+page.markdoc [hunk 7]
```diff
@@ -7,13 +7,13 @@ Databases are the largest organizational unit in Appwrite.
 Each database contains a group [collections](/docs/products/databases/collections).
 In future versions, different databases may be backed by a different database technology of your choosing.
 
-# Create a database in Console
+# Create in Console {% #create-db-in-console %}
```
Comment (gewenyu99) [inline src/routes/docs/products/databases/databases/+page.markdoc:10]:
```suggestion
# Create in Console {% #create-in-console %}
```

File: src/routes/docs/products/databases/databases/+page.markdoc [hunk 8]
```diff
@@ -7,13 +7,13 @@ Databases are the largest organizational unit in Appwrite.
...
 The easiest way to create a database using the Appwrite Console.
 You can create a database by navigating to the **Databases** page and clicking **Create database**.
 
 
 
-# Create a database using Server SDKs
+# Create using Server SDKs {% #create-db-in-sdk %}
```
Comment (gewenyu99) [inline src/routes/docs/products/databases/databases/+page.markdoc:16]:
```suggestion
# Create using Server SDKs {% #create-using-server-sdks %}
```

File: src/routes/docs/products/databases/relationships/+page.markdoc [hunk 9]
```diff
@@ -398,7 +398,7 @@ databases.createDocument(
 ```
 {% /multicode %}
 
-## Edge case behaviors
+## Edge case behaviors {% #edge-case %}
```
Comment (gewenyu99) [inline src/routes/docs/products/databases/relationships/+page.markdoc:401]:
```suggestion
## Edge case behaviors {% #edge-case-behaviors %}
```

File: src/routes/docs/products/storage/upload-download/+page.markdoc [hunk 10]
```diff
@@ -130,7 +130,7 @@ If you're not using an SDK, you can [learn more about REST API file handling](/d
 # InputFile {% #input-file %}
 Every language and platform handles file inputs differently. This section documents the expected input type of each SDK. Where applicable, Appwrite provides an `InputFile` class to accept multiple file sources, like paths, buffers, streams, or plain text.
 
-# Client SDKs
+# Client SDKs {% #client-sdk %}
```
Comment (gewenyu99) [inline src/routes/docs/products/storage/upload-download/+page.markdoc:133]:
```suggestion
# Client SDKs {% #client-sdks %}
```

File: src/routes/docs/products/storage/upload-download/+page.markdoc [hunk 11]
```diff
@@ -180,7 +180,7 @@ The Appwrite Apple SDK expects an `InputFile` class for file inputs.
 {% /tabsitem %}
 {% /tabs %}
 
-# Server SDK
+# Server SDK {% #server-sdk %}
```
Comment (gewenyu99) [inline src/routes/docs/products/storage/upload-download/+page.markdoc:183]:
```suggestion
# Server SDKs {% #server-sdks %}
```

File: src/routes/docs/tutorials/react/step-6/+page.markdoc [hunk 12]
```diff
@@ -21,7 +21,7 @@ Create a new collection with the following attributes:
 | title       | String | Yes      |
 | description | String | No       |
 
-# Ideas context
+# Ideas context {% #idea-context %}
```
Comment (gewenyu99) [inline src/routes/docs/tutorials/react/step-6/+page.markdoc:24]:
```suggestion
# Ideas context {% #ideas-context %}
```

File: src/routes/docs/tutorials/sveltekit/step-6/+page.markdoc [hunk 13]
```diff
@@ -21,7 +21,7 @@ Create a new collection with the following attributes:
 | title       | String | Yes      |
 | description | String | No       |
 
-# Add and remove methods
+# Add and remove methods {% #add-remove-methods %}
```
Comment (gewenyu99) [inline src/routes/docs/tutorials/sveltekit/step-6/+page.markdoc:24]:
```suggestion
# Add and remove methods {% #add-and-remove-methods %}
```

**Review by eldadfux — APPROVED** [review]

**Pull request merged**